server/src/service/session.rs

```diff
@@ -0,0 +1,35 @@
+// org.freedesktop.Secret.Session
+
+use crate::SESSION;
+use zbus::{dbus_interface, fdo, zvariant, ObjectServer, SignalContext};
+use zvariant::{ObjectPath, OwnedObjectPath};
+
+#[derive(Debug)]
+pub struct Session {
+    path: OwnedObjectPath,
+}
+
+#[dbus_interface(name = "org.freedesktop.Secret.Service")]
+impl Session {
+    pub async fn close(
+        &mut self,
+        #[zbus(object_server)] server: &zbus::ObjectServer,
+    ) -> fdo::Result<()> {
+        server.remove::<Self, &OwnedObjectPath>(&self.path).await?;
+        Ok(())
+    }
+}
+
+impl Session {
+    pub async fn new(path: OwnedObjectPath) -> Self {
+        Self { path: path.into() }
+    }
+
+    pub async fn path(&self) -> &OwnedObjectPath {
+        &self.path
+    }
+}
+
+pub async fn init_session_service() -> Session {
+    Session::new(ObjectPath::try_from("/").unwrap().into()).await
```

Comment:
If you're looking for an (Owned)ObjectPath that is just "/", you should use [(Owned)ObjectPath::default()](https://docs.rs/zvariant/3.15.0/src/zvariant/object_path.rs.html#118-122)

This also eliminates some more unnecessary `.unwrap()`s

Reply:
This is much cleaner approach. Thanks!